### PR TITLE
SISRP-39228 - Tasks Card Divider Horizontal Line Missing

### PR DESCRIPTION
**Pull request by redconfetti**
https://jira.berkeley.edu/browse/SISRP-39228

### CODE REVIEW

File: src/assets/templates/widgets/tasks.html [hunk 1]
```diff
@@ -85,71 +85,71 @@ <h2 class="cc-left">Tasks</h2>
...
-                    <div class="cc-left">
-                      <span class="cc-widget-task-section-incomplete-count">
-                        <i class="fa fa-bell" aria-hidden="true"></i>
-                        <span data-ng-bind="taskSection.tasks.incomplete.length"></span> incomplete
-                      </span>
+            <div class="cc-widget-tasks-section-wrapper" data-ng-class="{'cc-widget-tasks-section-wrapper-top-border': $first}">
+              <div class="cc-widget-tasks-section-header-incomplete">
+                <div class="cc-flex cc-flex-space-between cc-flex-align-center">
+                  <div>
+                    <div >
```
Comment (lyttam) [inline src/assets/templates/widgets/tasks.html]: @redconfetti There's a space in this <div> tag.  I see it was there before your change, so as long as this template renders OK in major browsers it's fine to leave it.

Comment (redconfetti) [reply src/assets/templates/widgets/tasks.html]: @lyttam - Great catch. I removed the space, and the unnecessary DIV.

**Comment by redconfetti** [comment]
@lyttam - I've revised the template to remove the space.

**Comment by nkutub** [comment]
LGTM @redconfetti

**Pull request merged**